packages/SwingSet/src/liveslots/collectionManager.js

```diff
@@ -890,8 +926,17 @@ export function makeCollectionManager(
    */
   function makeScalarBigWeakSetStore(
     label = 'weakSet',
-    { keySchema = M.scalar(), valueSchema = undefined, durable = false } = {},
+    {
+      keySchema = M.scalar(),
+      valueSchema = undefined,
+      durable = false,
+      fakeDurable = false,
+    } = {},
   ) {
+    assert(
+      !durable || !fakeDurable,
+      'durable and fakeDurable are mutually exclusive',
+    );
```

Comment:
Amount of duplicated code here is unfortunate. If there's a way to consolidate without making other things noticeably worse, it's probably worth it.